templates/user_profile_page.hbs

```diff
@@ -28,9 +28,9 @@
             </svg>
           </span>
         {{/if}}
-        <span class="edit-profile-button">
-          {{edit}}
-        </span>
+        <div class="user-profile-actions">
+          {{actions}}
```

Comment:
In this case, `actions` could take `className` as a parameter, instead of wrapping the button, which we did for legacy concerns.

Reply:
👍 
Would you still wrap it in a `div` then, or?

Reply:
No, since the helper already outputs a div

Reply:
I pushed an update. And on the corresponding HC branch.